micro_sam/sam_annotator/_widgets.py

```diff
@@ -16,6 +16,7 @@
 import z5py
 
 from qtpy import QtWidgets
+from qtpy.QtWidgets import QApplication, QDialog, QVBoxLayout, QHBoxLayout, QLabel, QPushButton, QMessageBox
```

Comment:
Let's not duplicate these imports. We import `qtpy.QtWidgets` in the line above, so you can use `QtWidgets.QApplication` etc. throughout the code.